sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/TestJsonData.scala

```diff
@@ -17,14 +17,15 @@
 
 package org.apache.spark.sql.execution.datasources.json
 
-import org.apache.spark.rdd.RDD
-import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.{Dataset, SparkSession, SQLContext}
+import org.apache.spark.sql.test.SharedSQLContext
 
-private[json] trait TestJsonData {
-  protected def spark: SparkSession
+private[json] trait TestJsonData extends SharedSQLContext {
+  val importContext = sqlContext
+  import importContext.implicits._
```

Comment:
I guess you want [`testImplicits._`](https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestUtils.scala#L61-L70)..